templates/cluster_resources.j2

```diff
@@ -90,6 +97,16 @@ primitive rsc_ip_{{ sid }}_ERS{{ ers_instance }} ocf:heartbeat:gcp-vpc-move-rout
 {%- else %}
 
 {%- if cloud_provider == "microsoft-azure" %}
+{%- if native_fencing %}
+property $id="cib-bootstrap-options" \
+    stonith-enabled="true" \
+    concurrent-fencing=true
+
+primitive rsc_azure_stonith_{{ sid }} stonith:fence_azure_arm \
+    params subscriptionId={{ grains['subscription_id'] }} resourceGroup={{ grains['resource_group_name'] }} tenantId={{ grains['tenant_id'] }} login={{ grains['fence_agent_app_id'] }} passwd="{{ grains['fence_agent_client_secret'] }}" pcmk_monitor_retries=4 pcmk_action_limit=3 power_timeout=240 pcmk_reboot_timeout=900 \
```

Comment:
Hey @yeoldegrove ,
I noticed that we cannot use the `grains` entries here. The grains are only coming from `ha-sap-deployments-formula`, but the project should work standalone.
We would need to replace this to use the pillar file entry (variables coming from `data`).

For that, I guess we will need to change the `ha-sap-terraform-deployments` project to modify the pillar file on the automatic folder.

You will need to refactor the change in the `saphanabootstrap-formula`, which, unfortunately I merged... (don't increase the version number, just apply the fix on the hana formula)

Reply:
@arbulu89 now pillars are used and some documentation is available here https://github.com/SUSE/habootstrap-formula/pull/84